nutrition_labels/predict.py

```diff
@@ -79,12 +102,17 @@ def predict_grants(config):
         help='Path to config file',
         default='configs/predict/2021.04.02.ini'
     )
+    parser.add_argument(
+        '--grants_data_path',
+        help='Path to grants data csv path to make predictions on',
+        default=None
+    )
 
     args = parser.parse_args()
 
     config = configparser.ConfigParser()
     config.read(args.config_path)
 
-    grant_ids, y_pred = predict_grants(config)
+    grant_ids, y_pred = predict_grants(config, args.grants_data_path)
```

Comment:
why don't you add this to the config?

Reply:
That was one of my comments in the previous PR. To enable us to predict on _any_ data, and not tie "predictions" to training, as we move along separating the functionalities.

Reply:
(actually you can still add in the config file, it's just that the argument takes precedence)